blocks/card-list-block/README.md

```diff
@@ -21,7 +21,10 @@ _Please provide a 1-2 sentence description of what the block is and what it does
 Outline any schema information requirements necessary to know for ths block
 
 ### ANS Fields
-- `Add all ANS fields used in the block`
+- `content_elements[x].websites[arcSite].website_url`
```

Comment:
Should this be in JSON?

Reply:
could be. I was going off the template that Robin had setup for the ticket:

author bio readme
```
### ANS fields
- `credits.by`
```

Reply:
So, I added this format because as a fusion developer, we always get the response back from content sources as a usable object. I thought it was easier to read/document which particular fields were being used by doing it this way. I personally think the readability is improved by having a simple list instead of writing out the JSON longhand, but I could be wrong!